ui.css

```diff
@@ -41,12 +41,23 @@
   display: flex;
 }
 
+#page-game[stage=playing] {
+    flex: 1 1 auto;
+    min-height: 0;
```

Comment:
It needs a min-height of 0? Does it get a negative height somehow?

Reply:
No, min-height:0 is needed to allow flex to shrink properly, otherwise the minimum size of a flex widget is implied to be it's min-content size.